PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskNewJetSubstructure.cxx

```diff
@@ -302,6 +304,10 @@ void AliAnalysisTaskNewJetSubstructure::UserCreateOutputObjects() {
     }
 
   fShapesVarNames[22] = "etaJet";
+  if(fDoTime){
+    fShapesVarNames[23] = "tauForm";
+    fShapesVarNames[24] = "tauFormMatch";
+  }
```

Comment:
I'm a little rusty on this task, but looking ahead a little bit, I think you're missing `tauDet`, which will be used when you're embedding